src/en/rezoscans/build.gradle

```diff
@@ -3,8 +3,7 @@ ext {
     extClass = '.RezoScans'
     themePkg = 'keyoapp'
     baseUrl = 'https://rezoscans.com'
-    overrideVersionCode = 0
-    isNsfw = false
```

Comment:
`isNsfw = false` makes it explicit the site is not NSFW. If whether the site is NSFW or not is ambiguous I guess removing it should be good.

Yes, functionally there are no changes. But I would argue it should be kept for documentation.